celery/app/autoretry.py

```diff
@@ -48,7 +48,7 @@ def run(*args, **kwargs):
                 if retry_backoff:
                     retry_kwargs['countdown'] = \
                         get_exponential_backoff_interval(
-                            factor=retry_backoff,
+                            factor=int(max(1.0, retry_backoff)),
```

Comment:
are we going to turn everything into 1 only or multiple of 1.0?

Reply:
All values less then 1 turn into 1 (except zero). For the other values, the behavior remains the same. I'm using 1.0 (not 1) just for linter.